libs/design/src/molecules/list/list/list.component.scss

```diff
@@ -0,0 +1,70 @@
+@import '../../../scss/daff-util';
+
+.daff-list {
+  $root: &;
+  display: block;
+  margin: 0;
+  padding: 0;
+  
+  &__subheader {
+    @include embolden();
+    @include uppercase();
+    font-size: 1rem;
+    padding: 0 0 25px 0;
+  }
+
+  &__item {
+    display: block;
+    padding: 8px 0;
```

Comment:
Margin, not padding.

Reply:
I disagree. What if I want to add a background color hover effect on a `list-item`?

Reply:
Hrm. Then I don't think that [daff-list-item] should be an attribute. It should be a element. We don't style the content of list items in lists, we merely style the element and a few distinct elements like subheader.